doc/i18n.md

```diff
@@ -0,0 +1,72 @@
+## Using the I18n module
+
+**This document does not apply to apps that are migrated from v1 of the Zendesk
+App Framework.**
+
+The I18n module provides a helper called `I18n.t` that you can use to localize
+your application.
```

Comment:
Should we mention the intention of the module is to assist migration, and people should consider using a premade i18n library for newly developed apps?